app/src/main/java/com/hedvig/app/feature/onbarding/OnboardingModel.kt

```diff
@@ -0,0 +1,26 @@
+package com.hedvig.app.feature.onbarding
+
+import androidx.recyclerview.widget.DiffUtil
+import com.hedvig.android.owldroid.graphql.ChoosePlanQuery
+
+sealed class OnboardingModel {
+
+    data class Bundle(
+        val selected: Boolean,
+        val embarkStory: ChoosePlanQuery.EmbarkStory
+    ) : OnboardingModel()
+
+    object Error : OnboardingModel()
+}
+
+class OnboardingDiffUtilCallback : DiffUtil.ItemCallback<OnboardingModel>() {
+    override fun areItemsTheSame(oldItem: OnboardingModel, newItem: OnboardingModel): Boolean {
+        if (oldItem is OnboardingModel.Bundle && newItem is OnboardingModel.Bundle) {
+            return true
+        }
```

Comment:
```suggestion
        if (oldItem is OnboardingModel.Bundle && newItem is OnboardingModel.Bundle) {
            return true
        }
```
I don't think this holds: the items are the same only if the inner `.embarkStory.name` is the same, not any old `OnboardingModel.Bundle` 😬